src/ITensorNetworks/peps.jl

```diff
@@ -81,10 +81,36 @@ broadcast_inner(A::PEPS, B::PEPS) = mapreduce(v -> v[], +, A.data .* B.data)
 
 ITensors.prime(P::PEPS, n::Integer=1) = PEPS(map(x -> prime(x, n), P.data))
 
+# prime a PEPS with specified indices
+function ITensors.prime(indices::Array{<:Index,1}, P::PEPS, n::Integer=1)
```

Comment:
Would be nice to implement a more direct `prime(siteinds, ::PEPS, ...)` function to complement `prime(linkinds, ::PEPS, ...)`. I'll add that as an issue.